examples/model_selection/plot_precision_recall.py

```diff
@@ -139,20 +139,32 @@
 plt.legend(loc="lower left")
 plt.show()
 
-# Plot Precision-Recall curve for each class
+# Plot Precision-Recall curve for each class and iso-f1 curves
 plt.clf()
+f_scores = np.linspace(0.1, 0.9, num=9)
+lines = []
+labels = []
+for F in f_scores:
+    x = R = np.linspace(0.01, 1)
+    y = F * R / (2 * R - F)
```

Comment:
also you could use `f_score` instead of `f` maybe? And reuse `x` instead of `R`?